packages/tasks/src/model-libraries.ts

```diff
@@ -241,6 +241,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://huggingface.co/docs/hub/keras",
 		snippets: snippets.keras,
 		filter: true,
+		countDownloads: `path:"config.json"`,
```

Comment:
```suggestion
		countDownloads: `path:"config.json" OR extension:"keras"`,
```

(disclaimer: i haven't checked the actual proper syntax)

Reply:
Also we will need to update https://huggingface.co/docs/hub/keras and move the current version to https://huggingface.co/docs/hub/tf-keras

Reply:
The extension is `.h5` (`keras` was the zip, we upload unzipped)

Reply:
For what we discussed, @julien-c 's suggestion is too tag repos that have a `.keras` file as `keras` => it's not the structure we want to promote but it's still good to support it. In that case, we should count downloads with the .keras extension.

Counting `.h5` file is not needed since the `.h5` file will always be downloaded at the same time as the `config.json` file.

Reply:
done in https://github.com/huggingface/huggingface.js/pull/774/commits/f7500bc3205d670bd06c2e8f0b3212c973e7ea8e